apps/src/StudioApp.js

```diff
@@ -196,10 +200,9 @@ var StudioApp = function () {
   this.wireframeShare = false;
 
   /**
-   * Redux store that might be provided by the app. Initially give it an empty
-   * interface so that we can assume existence.
+   * Redux store that might be provided by the app.
```

Comment:
Hah. Updated the comment, but not enough. You're right - StudioApp always creates the store, the app may (or may not) provided app specific reducers.